threedi_schema/application/schema.py

```diff
@@ -127,6 +129,27 @@ def _get_epsg_data(self) -> Tuple[Optional[int], str]:
                     return srids[0], f"{model.__tablename__}.geom"
         return None, ""
 
+    def _get_dem_epsg(self, raster_path=None) -> int:
```

Comment:
Please add a type hint for raster_path to make clear that this should be a string and not a Path instance.

```suggestion
    def _get_dem_epsg(self, raster_path: str=None) -> int:
```